extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/values.yaml

```diff
@@ -24,7 +24,7 @@ aissemble-quarkus-chart:
           name: spark-application-values-files
 
   configMap:
-    name: supplemental-quarkus-config
+    name: pipeline-invocation-quarkus-config
```

Comment:
Q: Presumably we don't have to worry about migrations because any down stream project would have had to fix this already or they only have one Quarkus deployment?

Reply:
Correct